.github/workflows/publish.yml

```
@@ -0,0 +1,63 @@
+name: publish
+on: push
+
+env:
+  PYTHON_VERSION: 3.10.10
+  POETRY_VERSION: 1.6.1
+
+jobs:
+  pypi-publish:
+    name: Upload release to PyPI
```

```suggestion
    name: Publish Python 🐍 distribution 📦 to PyPI
```

Just to keep both job names :sparkles: fun :sparkles: